scenarios/glue_privesc/terraform/provider.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
```

Comment:
Missing version constraint for provider "local" in `required_providers`
 
~~Missing version constraint for provider "template" in `required_providers`~~ (see other note)
 
It does not look like the time provider is used